Kafka Consumer to test sample

suganyasuven opened the pull request:
## Purpose
> To view the output on the kafka-server which were sent through kafka-server. It is a simple server written in siddhi.

appellj commented:
In the prereqs in the Kafka sample, we say this:

1) Setup Kafka
 +        * Kafka libs to be added and converted to OSGI from {KafkaHome}/libs are as follows
 +            * kafka_2.10-0.9.0.1.jar
 +            * kafka-clients-0.9.0.1.jar
 +            * metrics-core-2.2.0.jar
 +            * scala-library-2.10.5.jar
 +            * zkclient-0.7.jar
 +            * zookeeper-3.4.6.jar
 +        *  Add the OSGI converted kafka libs to {WSO2SPHome}/lib
 +        *  Add the kafka libs to {WSO2SPHome}/samples/sample-clients/lib

This doesn't explain how to convert them to OSGi. Are you supposed to copy this list of JARs to the {WSO2SPHome}/samples/sample-clients/lib folder and then convert them to OSGi and copy those converted files to {WSO2SPHome}/lib? If so, how do you convert them?

suganyasuven commented:
Added the requested steps in the documentation.


On Sat, Dec 16, 2017 at 6:17 AM, Jackie Wheeler <notifications@github.com>
wrote:

> In the prereqs in the Kafka sample, we say this:
>
>    1. Setup Kafka
>
>
>    -
>
>       * Kafka libs to be added and converted to OSGI from {KafkaHome}/libs are as follows
>
>    -
>
>           * kafka_2.10-0.9.0.1.jar
>
>    -
>
>           * kafka-clients-0.9.0.1.jar
>
>    -
>
>           * metrics-core-2.2.0.jar
>
>    -
>
>           * scala-library-2.10.5.jar
>
>    -
>
>           * zkclient-0.7.jar
>
>    -
>
>           * zookeeper-3.4.6.jar
>
>    -
>
>       *  Add the OSGI converted kafka libs to {WSO2SPHome}/lib
>
>    -
>
>       *  Add the kafka libs to {WSO2SPHome}/samples/sample-clients/lib
>
>
> This doesn't explain how to convert them to OSGi. Are you supposed to copy
> this list of JARs to the {WSO2SPHome}/samples/sample-clients/lib folder
> and then convert them to OSGi and copy those converted files to
> {WSO2SPHome}/lib? If so, how do you convert them?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wso2/product-sp/pull/256#issuecomment-352146584>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/Ac1PnaMOyXr-WfsytPG8sAKAtJG74Doxks5tAxMqgaJpZM4RBirm>
> .
>
